### PR TITLE
Environment variables

### PR DESCRIPTION
**Pull request by aguillenv**
Fixes: https://github.com/alantech/meta/issues/165

### CODE REVIEW

File: avm/src/daemon/linux.rs [hunk 1]
```diff
@@ -73,6 +73,7 @@ async fn post_v1_scale(cluster_id: &str, agz_b64: &str, deploy_token: &str, fact
   let pwd = var("PWD").unwrap();
   let dockerfile = read(format!("{}/Dockerfile", pwd));
   let app_tar_gz = read(format!("{}/app.tar.gz", pwd));
+  let env_file = read(format!("{}/.anycloud.env", pwd));
```
Comment (depombo) [inline avm/src/daemon/linux.rs:76]: per https://github.com/alantech/deploy/pull/54/files#r594423152, we should check that it exists like the other 2 files in the line below

File: avm/src/daemon/linux.rs [hunk 2]
```diff
@@ -73,22 +73,19 @@ async fn post_v1_scale(cluster_id: &str, agz_b64: &str, deploy_token: &str, fact
   let pwd = var("PWD").unwrap();
```
Comment (cdmistman) [inline avm/src/daemon/linux.rs:73]: nit: <https://doc.rust-lang.org/std/env/fn.current_dir.html> will work on all platforms rust supports

File: avm/src/daemon/linux.rs [hunk 3]
```diff
@@ -73,22 +73,19 @@ async fn post_v1_scale(cluster_id: &str, agz_b64: &str, deploy_token: &str, fact
...
+    "clusterId": cluster_id,
+    "agzB64": agz_b64,
+    "deployToken": deploy_token,
+    "clusterFactor": factor,
+  });
+  if dockerfile.is_ok() && app_tar_gz.is_ok() {
+    scale_body.as_object_mut().unwrap().insert(format!("DockerfileB64"), json!(base64::encode(dockerfile.unwrap())));
+    scale_body.as_object_mut().unwrap().insert(format!("appTarGzB64"), json!(base64::encode(app_tar_gz.unwrap())));
+  }
+  if env_file.is_ok() {
```
Comment (cdmistman) [inline avm/src/daemon/linux.rs:87]: nit: `if let` to avoid unwrapping below

**Review by dfellis — APPROVED** [review]

**Review by depombo — APPROVED** [review]
Just have to fix merge conflicts now, but looks good!

**Review by cdmistman — APPROVED** [review]

**Pull request merged**